.github/workflows/ci.yml

```
@@ -45,7 +45,9 @@ jobs:
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 6.0.x |
+            7.0.x
+          include-prerelease:true
```

Apparently this is a syntax error not to have a space too
```suggestion
          include-prerelease: true
```